src/vodozemac/pk_encryption.rs

```diff
@@ -102,7 +67,43 @@ impl PkMessage {
     }
 }
 
+/// The base64-encoded variant of a {@link PkMessage}.
+///
+/// This can be useful if the encrypted message should be put into JSON.
+#[wasm_bindgen(getter_with_clone)]
+pub struct Base64EncodedPkMessage {
+    /// The base64-encoded ciphertext.
+    pub ciphertext: String,
+    /// The base64-encoded message authentication code (MAC).
+    pub mac: String,
+    /// The base64-encoded ephemeral public key.
+    #[wasm_bindgen(js_name = "ephemeralKey")]
+    pub ephemeral_key: String,
+}
+
+#[wasm_bindgen]
+impl Base64EncodedPkMessage {
+    /// Creates a new base64-encoded encrypted message from its parts.
+    #[wasm_bindgen(constructor)]
+    pub fn new(ciphertext: &str, mac: &str, ephemeral_key: &str) -> Self {
+        Self {
+            ciphertext: ciphertext.to_owned(),
+            mac: mac.to_owned(),
+            ephemeral_key: ephemeral_key.to_owned(),
+        }
+    }
+}
+
 /// A class representing a public-key encryption instance.
+///
+/// This implements the encryption part of the
+/// `m.megolm_backup.v1.curve25519-aes-sha2` algorithm described in the Matrix
+/// [spec].
+///
+/// More details can be found in the official [vodozemac documentation].
+///
+/// [spec]: https://spec.matrix.org/v1.11/client-server-api/#backup-algorithm-mmegolm_backupv1curve25519-aes-sha2
+/// [vodozemac documentation]: https://docs.rs/vodozemac/latest/vodozemac/pk_encryption/
```

Comment:
I don't think this link style works with tsdoc. You'll need [`@link`](https://tsdoc.org/pages/tags/link/).

Reply:
Could probably also do with an `@see {@link PkDecryption}`.

Reply:
likewise below

Reply:
Did so in https://github.com/matrix-org/matrix-rust-sdk-crypto-wasm/pull/209/commits/f1900d92feba6a07f7209915075e11b26bd07f7f.